Lib/unittest/test/testmock/test_sealable.py

```diff
@@ -0,0 +1,185 @@
+import unittest
+from unittest import mock
+
+
+class SampleObject(object):
+    def __init__(self):
+        self.attr_sample1 = 1
+        self.attr_sample2 = 1
+
+    def method_sample1(self):
+        pass
+
+    def method_sample2(self):
+        pass
+
+
+class TestSealable(unittest.TestCase):
+    """Validates the ability to seal a mock which freezes its spec"""
+
+    def test_attributes_return_more_mocks_by_default(self):
+        m = mock.Mock()
+
+        assert isinstance(m.test, mock.Mock)
```

Comment:
Please use standard unittest assert methods, `assertIsInstance` in this case.

Reply:
done